lib/samson/hooks.rb

```diff
@@ -18,6 +19,7 @@ class UserError < StandardError
       :project_permitted_params,
       :before_deploy,
       :after_deploy_setup,
+      :after_deploy_create, # for external plugin
```

Comment:
you could make after_deploy_setup also pass the deploy 

after_deploy_setup and after_deploy_save would be called at almost the same time, that's why I was thinking we don't need another hook ... store the attributes in the deploy and then use after_deploy_setup and read it back from the deploy